documentation/docs/examples/image360.mdx

```diff
@@ -0,0 +1,97 @@
+---
+id: image360
+title: 360 Images
+description: Using and displaying 360 images using Reveal
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+360 images is a common way to augment the 3D experience of your application.
+A 360 image is a controllable panoramic image that surrounds the original point from which the shot was taken. 
+360 images simulate being in the shoes of a photographer and looking around to the left, right, up and down as desired as well as sometimes zooming.
+
+The `Cognite3DViewer` entrypoint exposes the following methods related to 360 images.
+
+```ts
+add360ImageSet(
+  datasource: 'events', 
+  eventFilter: { [key: string]: string }, 
+  add360ImageOptions?: AddImage360Options
+): Promise<Image360[]>;
+
+remove360Images(...image360Entities: Image360[]): Promise<void>:
+
+enter360Image(image360: Image360): Promise<void>:
+
+exit360Image(): void
+```
+
+To visualize 360 images the data must have been previously ingested into [Cognite Data Fusion](https://www.cognite.com/en/product/cognite_data_fusion_industrial_dataops_platform).
+More info on the ingestion process can be found [here](https://docs.cognite.com/cdf/3d/guides/360images_upload/).
+
+Once the data is ingested, a set of 360 images can be added to the viewer with `viewer.add360ImageSet(...)`.
+`datasource` describes the source data type used for storing the 360 images.
+At this time, the only valid `datasource` is `'events'`, but you can expect more to be added in the future.
+The `eventFilter` property describes the filter that is used when fetching data from the data source.
+This is a generic key-value pair and can point to any metdata that was set during ingestion.
+
+The `AddImage360Options` is used for correcting / adding transformations to the set of 360 images and has the following declaration:
+```ts
+type AddImage360Options = {
+  /**
+   * An optional transformation which will be applied to all 360 images that are fetched.
+   */
+  collectionTransform?: THREE.Matrix4;
+  /**
+   * Set this to false if the 360 images' rotation is not pre-multiplied to fit the given model.
+   * @default true
+   */
+  preMultipliedRotation?: boolean;
+};
+```
+
+:::note
+The 360 images should be ingested with proper transformation data, and one should only rely on `AddImage360Options` when failing to do so or when using the same 360 images for multiple 3D models with different coordinate system.
+If you find that the data does not line up correctly, please try to update the source data instead adding the optional transformation.
+:::
+
+If the data is ingested similar to the guide referenced earlier, adding a set of 360 images can look like this:
+```ts
+viewer.add360ImageSet('events', { site_id: 'site-S01-area-a03' });
+```
+
+Once added to the viewer you will see a white circle representing each 360 image, here is an example of what it can look like:
+<img src={useBaseUrl('/img/image360_1.png')} alt="AxisViewTool" />
```

Comment:
Should we have an image showing the view from within a 360 image, with the other icons visible etc.?